config/environments/production.rb

```diff
@@ -55,15 +56,15 @@
   # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
 
   # Use a different cache store in production.
-  # config.cache_store = :mem_cache_store
+  config.cache_store = :mem_cache_store
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
   # config.action_controller.asset_host = 'http://assets.example.com'
 
   # Ignore bad email addresses and do not raise email delivery errors.
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
-  config.action_mailer.default_url_options = { host: "lib-plum1.princeton.edu" } # FIXME: replace with IU equivalent?
+  config.action_mailer.default_url_options = { host: "localhost" } # FIXME: replace with IU equivalent?
```

Comment:
This can be added to the list of productionizing done via merge from a private secrets branch in the same way all the other configuration will be done.